demos/smart_classroom_demo/cpp_gapi/include/initialize.hpp

```diff
@@ -12,6 +12,7 @@
 #include <vector>
 
 #include <opencv2/gapi/infer/ie.hpp>
+#include <opencv2/gapi/infer/ov.hpp>
```

Comment:
`Build Clang Demos (Ubuntu 20.04):` `/opt/home/sys_k8sworker/ci-ov-omz/workspace/omz/precommit/build_clang_demos_ubuntu20/src/demos/smart_classroom_demo/cpp_gapi/include/initialize.hpp:15:10: fatal error: 'opencv2/gapi/infer/ov.hpp' file not found`

I believe upgrading demos to G-API with ov 2.0 was going to be a separate activity

Reply:
Discussed locally. I disabled G-API build with low OpenCV version. Also enabled some tests in github-actions according to https://github.com/openvinotoolkit/open_model_zoo/pull/3878